lib/bundle_update_interactive/cli/options.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "optparse"
+
+module BundleUpdateInteractive
+  class CLI::Options
+    class << self
+      def parse(argv=ARGV)
+        options = new
+        remaining = build_parser(options).parse!(argv.dup)
+        raise Error, "update-interactive does not accept arguments. See --help for available options." if remaining.any?
+
+        options.freeze
+      end
+
+      private
+
+      def build_parser(options) # rubocop:disable Lint/UnusedMethodArgument
```

Comment:
🗒️ In the future, when we add command-line flags, those flags could be represented as attributes on the `options` object. E.g.

```ruby
parser.on("--quiet", "Silence progress output") do
  options.quiet = true
end
```

Then the CLI can access them:

```ruby
options = CLI::Options.parse(ARGV)
options.quiet?
```